src/test/scala/nl/knaw/dans/easy/bag2deposit/ddm/ProvenanceSpec.scala

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.bag2deposit.ddm
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class ProvenanceSpec extends AnyFlatSpec with Matchers {
+  "Provenance" should "show diff" in {
+    val ddmIn =
+      <ddm>
+        <ddm:profile>
+          <dc:title>Rapport 123</dc:title>
+        </ddm:profile>
+        <ddm:dcmiMetadata>
+            <dc:title>blabla</dc:title>
+            <dc:title>Rapport 456</dc:title>
+            <dc:title>Transect-rapport 2859: Een Archeologisch Bureauonderzoek. Ellecom, glasvezeltracé Eikenstraat, Gemeente Rheden (GD).</dc:title>
+            <dc:title>rabarbera</dc:title>
+            <dc:title>Archeologische Berichten Nijmegen – Briefrapport 21</dc:title>
```

Comment:
make a test for the alternatives?
```suggestion
            <dc:alternative>Archeologische Berichten Nijmegen – Briefrapport 21</dc:alternative>
```